### PR TITLE
Search Results

### PR DESCRIPTION
**Pull request by OxanaMl**
## Description
Added Search Results Page that will populate a list of hotels based on different criteria such as destination, checkin date, checkout date, number of guests, and number of rooms. 

*** Update: Merge and integrated search results page with the flow of the app. User should be able to start on the homepage and search a destination. Takes you to the search results page where you can see the hotels results and repeat the process in terms of searching. Researching on the search results  page just reloads the page with the new results. 

## Acceptance Criteria
The page is responsive, console has no errors.

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

### Before

image

### After
image

## Testing Steps / QA Criteria
Start on the Homepage, search for a destination, review the list of hotels, check to see if hotel image, name and other details display properly, check for page responsiveness. When clicked on "See Availability" button, it should bring you to the Hotel Detail page.

### CODE REVIEW

**Review by marinacod — COMMENTED** [review]
Increase the size of destination

**Review by madebymiguel — CHANGES_REQUESTED** [review]
Looks good overall. 

- Increase font size of city and number of hotels.  The "loading component" make it as a separate component/file. 
- Decrease margin top/bottom on review score

**Review by marinacod — APPROVED** [review]

**Review by madebymiguel — APPROVED** [review]
Looks good!

**Review by marinacod — APPROVED** [review]

**Review by marinacod — APPROVED** [review]

File: src/App.test.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
-import { render, screen } from '@testing-library/react';
-import App from './App';
+import { render, screen } from "@testing-library/react";
```
Comment (alton-michaux) [inline src/App.test.js]: We can probably remove this file for now

**Review by alton-michaux — APPROVED** [review]
Hey @OxanaMl nice work, search results page looks great. 

I'm going to go ahead and approve, but we should probably remove the test file before merging with dev. 
- It's not going to hurt anything, but we should probably just remove it unless we plan on trying to get to 100% test coverage (which is probably a little too much to take on at this point with the deadline approaching).

File: src/App.test.js (hunk 1 above)
Comment (OxanaMl) [reply src/App.test.js]: Hey @alton-michaux, thanks for your feedback! The test file has been removed.

**Review by madebymiguel — APPROVED** [review]
Looks good

**Pull request merged**